DNN Platform/Modules/DigitalAssets/ClientScripts/dnn.DigitalAssets.js

```diff
@@ -2573,21 +2573,51 @@ dnnModule.digitalAssets = function ($, $find, $telerik, dnnModal) {
         }
     }
 
+    function listItemsToDelete(itemsToDelete, outputStr) {
+        for (var y = 0; y < itemsToDelete.length; y++) {
+            var delimiter = ", ";
+            if (y == (itemsToDelete.length - 1)) {
+                delimiter = "";
+            }
+
+            outputStr += itemsToDelete[y].ItemName + delimiter;
+        }
+        return outputStr;
+    }
+
     function confirmDeleteItems(items, parentFolderId, mappedSubfoldersCount) {
         var folderAndFileText = selectionText(items);
         var dialogTitle = resources.deleteTitle.replace('[ITEMS]', folderAndFileText);
         var dialogText = resources.deleteConfirmText.replace('[ITEMS]', folderAndFileText);
 
         var dialogNote = "";
-        var dialogHeight = 190;
+        var dialogHeight = 260;
         if (mappedSubfoldersCount > 0) {
             dialogNote = mappedSubfoldersCount == 1 ? resources.deleteConfirmWithMappedSubfolderText.replace('[COUNT]', mappedSubfoldersCount)
                                                     : resources.deleteConfirmWithMappedSubfoldersText.replace('[COUNT]', mappedSubfoldersCount);
             dialogNote = "<p class='dialogNote'>" + dialogNote + "</p>";
-            dialogHeight = 230;
+            dialogHeight = 280;
+        }        
+        var foldersToDelete = [];
```

Comment:
I would suggest to encapsulate this code into a function. Maybe called getDeletionDialogText or getDeleteItemsDialogText with the items as input parameter.